README.md

```diff
@@ -331,8 +376,9 @@ Thus, the possible options are:
 - `duration`, like the `show()` option
 - `multiple`, like the `show()` option
 - `newestOnTop`, like the `show()` option
-
-TODO([#11](https://github.com/jackbsteinberg/std-toast/issues/11)): How should actions be handled?
+- `action`, a string or element.
```

Comment:
should we specify HTMLElement here? Or should we get more specific and only allow `<a>` and `<button>` elements like we imply above?

Reply:
`Element` is a good idea